src/DepthImageToLaserScanROS.cpp

```diff
@@ -103,6 +102,6 @@ void DepthImageToLaserScanROS::depthCb(const sensor_msgs::msg::Image::SharedPtr
   }
   catch (std::runtime_error& e)
   {
-    ROS_ERROR("Could not convert depth image to laserscan: %s\n", e.what());
+    RCUTILS_LOG_ERROR("Could not convert depth image to laserscan: %s", e.what());
```

Comment:
feel free to put this back to throttled as it is in ROS1

Reply:
Will do.